firebase.json

```diff
@@ -8,9 +8,7 @@
       "cd packages/functions && rushx build",
       "sed -i '/\"workspace:\\*/d\"' \"$RESOURCE_DIR/package.json\""
     ],
-    "postdeploy": [
-      "sed -i -e \\'s/  .dependencies.: {/  \"dependencies\": {\\n    \"@eisbuk\\/shared\": \"1.0.0\",/\\' \"$RESOURCE_DIR/package.json\""
-    ],
+    "postdeploy": ["git restore packages/functions/package.json"],
```

Comment:
TIL about `git restore`! It's been around [for three years now](https://github.blog/2019-08-16-highlights-from-git-2-23/#experimental-alternatives-for-git-checkout), but I hadn't cought up yet. Thanks!

Reply:
Np. I use `git restore --staged <staged-file-or-pattern>` to unstage the file or in this context to restore an unstaged file to the version of the file on the last commit. 

And I always mix it up with `git reset` (now only in parlance and in typing, in the past I would also mix up the functionalities)